backend/src/test/java/de/bund/digitalservice/ris/caselaw/adapter/PublicPortalTransformerTest.java

```diff
@@ -0,0 +1,652 @@
+                 </akn:embeddedStructure>
+              </akn:block>
+               """),
+        Arguments.of(
+            "'otherLongText' (Sonstiger Langtext)",
+            """
+               <akn:block name="Sonstiger Langtext">
+                 <akn:embeddedStructure>
+                    <akn:p>otherLongText test</akn:p>
+                 </akn:embeddedStructure>
+               </akn:block>
+               """),
+        Arguments.of(
+            "'tenor' (Tenor)",
+            """
+               <akn:block name="Tenor">
+                   <akn:embeddedStructure>
+                      <akn:p>tenor test</akn:p>
+                   </akn:embeddedStructure>
+                </akn:block>
+               """),
+        // Fixme: should be included -->
+        //        Arguments.of(
+        //            "'participatingJudges' (Mitwirkende Richter)",
+        //            """
+        //               <akn:introduction>
+        //                  <akn:block name="Mitwirkende Richter">
+        //                     <akn:embeddedStructure>
+        //                        <akn:p>participating judge test</akn:p>
+        //                     </akn:embeddedStructure>
+        //                  </akn:block>
+        //               </akn:introduction>
+        //               """),
+        // ShortTexts/Kurztexte -->
+        Arguments.of(
+            "'guidingPrinciple' (Leitsatz)",
+            """
+               <akn:motivation>
+                 <akn:p>guidingPrinciple test</akn:p>
+               </akn:motivation>
+               """),
+        Arguments.of(
+            "'headline' (Titelzeile)",
+            """
+                <akn:header>
+                   <akn:p>headline test</akn:p>
+                </akn:header>
+               """));
+  }
+
+  static Stream<Arguments> testCasesForExcludedAttributes() {
+    return Stream.of(
+        Arguments.of(
+            "'decisionNames' (Entscheidungsnamen)",
+            """
+            <ris:decisionNames>
+               <ris:decisionName>decisionName test</ris:decisionName>
+            </ris:decisionNames>
+               """),
+        Arguments.of(
+            "'keywords' (Schlagworte)",
+            """
+        <akn:classification source="attributsemantik-noch-undefiniert">
+            <akn:keyword dictionary="attributsemantik-noch-undefiniert"
+                         showAs="attributsemantik-noch-undefiniert"
+                         value="keyword test"/>
+         </akn:classification>
+            """),
+        Arguments.of(
+            "'headnote' (Orientierungssatz)",
+            """
+                    <akn:block name="Orientierungssatz">
+                       <akn:embeddedStructure>
+                          <akn:p>headNote test</akn:p>
+                       </akn:embeddedStructure>
+                    </akn:block>
+                    """),
+        Arguments.of(
+            "'inputType' (EingabeTyp)",
```

Comment:
the original name is "Eingangsart" (I just fixed it in the google sheet as well)